util/new_keyboard.py

```diff
@@ -0,0 +1,148 @@
+#!/usr/bin/env python3
+
+import argparse
+from distutils.dir_util import copy_tree
+import fileinput
+import os
+import re
+import shutil
+import subprocess
+import sys
+
+def print_error(message):
+    print("[\033[0;91mERROR\033[m] " + message, file=sys.stderr)
+
+def prompt(message, default=""):
+    """
+    Asks the user for input, showing the default value, if any, in brackets.
+    """
+    if default:
+        message += f" [{default}]"
+    value = input(message + ": ")
+    return value if value else default
+
+def get_git_username():
+    """
+    Returns the username configured in git config, if any. Defaults to "qmk".
+    """
+    git = shutil.which("git")
+    if git:
+        devnull = open(os.devnull, 'w')
+        is_git_repo = (subprocess.call([git, "rev-parse", "--git-dir"], stdout=devnull, stderr=devnull) == 0)
+        if is_git_repo:
+            git_username = subprocess.check_output([git, "config", "--get", "user.name"]).decode("utf-8").rstrip('\r\n')
+            if git_username:
+                return git_username
+    return None
+
+def copy_template(keyboard_dir, keyboard_name, keyboard_type):
+    """
+    Copies the template files to the new keyboard's directory.
```

Comment:
```suggestion
    """Copy template files to the new keyboard's directory."""
```
As per PEP 257 and the above comments.